syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -929,17 +924,26 @@ def share(
             if self.type() == "torch.FloatTensor":
                 raise TypeError("FloatTensor cannot be additively shared, Use fix_precision.")
 
-            shared_tensor = (
-                syft.AdditiveSharingTensor(
-                    protocol=protocol,
-                    field=field,
-                    dtype=dtype,
-                    crypto_provider=crypto_provider,
-                    owner=self.owner,
-                )
-                .on(self.copy(), wrap=False)
-                .share_secret(*owners)
-            )
+            else:
+                if self.type() == "torch.FloatTensor":
```

Comment:
this condition is not reachable, you have already checked it previously.

Reply:
Sorry, fixed in latest commit